RELEASE-NOTES.txt

```diff
@@ -3,6 +3,7 @@
 17.9
 -----
 - [Internal] Payments: Updated StripeTerminal pod to 3.3.1 [https://github.com/woocommerce/woocommerce-ios/pull/12078]
+- Update Woo Purple color palette [https://github.com/woocommerce/woocommerce-ios/pull/12330]
```

Comment:
There's no merge conflict, but I think we want to add this for release 18.0 in the latest trunk now